test/4.x/app.js

```diff
@@ -128,17 +129,16 @@ app.get('/escape', function(req, res) {
 });
 
 app.get('/locals', function(req, res) {
-  res.locals.person = 'Alan';
```

Comment:
Removing this line is no longer resting particular functionality. `app.locals` is different than `res.locals`